pkg/sdk/README.md

```diff
@@ -0,0 +1,5 @@
+# Mainflux SDK kits
+
+This directory contains drivers for Mainflux HTTP API. Drivers do system administration (provisioning) and messaging. They are written in different languages in order to enable the faster application development in the respective language.
+
+Currently, there is only a fully developed SDK in Go. SDK's in Python and JavaScript are on the road map and are currently in development.
```

Comment:
I would remove this sentence, as once other SDK is merged, we'll forget to remove it.